src/main/java/core/basesyntax/dao/impl/CommentDaoImpl.java

```diff
@@ -40,8 +40,7 @@ public Comment create(Comment comment) {
     public Comment get(Long id) {
```

Comment:
how about others get() methods in other classes?

Reply:
done